src/vs/workbench/contrib/notebook/common/notebookCommon.ts

```diff
@@ -956,6 +956,7 @@ export const NotebookSetting = {
 	outputScrollingDeprecated: 'notebook.experimental.outputScrolling',
 	outputScrolling: 'notebook.output.scrolling',
 	textOutputLineLimit: 'notebook.output.textLineLimit',
+	LinkifyOutputFilepaths: 'notebook.output.linkifyFilepaths',
```

Comment:
Should it be `notebook.output.linkifyFilePaths` like how we name other settings (`breadcrumbs.filePath`, `files.dialog.defaultPath`)?